.github/workflows/ci.yml

```diff
@@ -143,7 +143,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: ['3.8', '3.9']
```

Comment:
Which Python version should we test here?

I have done some tests with the `continue-on-error` setting, but while the workflow run does not fail, each job is still reported as failing on the PR checks, which I think is quite confusing. I think it's better to keep only passing Python versions.

Reply:
I have kept only Python 3.8 in `setup.py`'s classifiers and in the CI, here and in all the other non-shared components